### PR TITLE
Display all subcategories to the page display all phrases to the page

### LINKED ISSUE #6: As a user - I should be able to click on the hello greeting and hear a sound out of the speakers.

### PR DESCRIPTION
**Pull request by Josephhaefling**
## Pull request checklist
Please check if your PR fulfills the following requirements:
- [ ] Lint (`npm run lint`) has passed locally and any fixes were made for failures
## Pull request type
Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Testing
- [ ] Other (please describe): 
## What is the current behavior?
- Previously a user could not click on the bank or party category and see their subcategories 
## What is the new behavior?
-  User can no click on the bank or party category and see the subcategories (Greetings, transactions) and by clicking on the subcategories they can see the phrases inside of each subcategory.
## Does this introduce a breaking change?
- [x] Yes
- [ ] No
Sound does not seem to be working.

### CODE REVIEW

**Review by jeremiahblackol — APPROVED** [review]
This made life easier

**Review by Josephhaefling — COMMENTED** [review]
Cool, we have sound!

File: src/Components/PhrasesPage/usePhrasePage.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import Phrase from '../Phrase/Phrase'
 function usePhrasePage(id, name, socialSettings) {
     const currentSetting = socialSettings.find(socialSetting => socialSetting.title === name)
     const subSettingPhrases = currentSetting.phrases.filter(phrase => phrase.tags[0] === id)
-    return subSettingPhrases.map(phrase => <Phrase categoryName={phrase} />
+    return subSettingPhrases.map(phrase => <Phrase categoryName={phrase}  phraseName={phrase.expression}  />
```
Comment (jeremiahblackol) [inline src/Components/PhrasesPage/usePhrasePage.js:7]: good job getting the data moving

**Review by jeremiahblackol — APPROVED** [review]

File: src/Components/SubCategoriesPage/useSubCategoriesPage.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import SubCategory from '../SubCategory/subCategory'
+import { Link } from 'react-router-dom'
+
+
```
Comment (jdvolk) [inline src/Components/SubCategoriesPage/useSubCategoriesPage.js:5]: looks great

File: src/Components/SubCategory/subCategory.js [hunk 3]
```diff
@@ -1,20 +1,22 @@
...
+    const { categoryName, phrases} = props    
+
     return(
         <section className='sub-category-container'>
-        <Emoji className="sub-category-emoji"/>
-            <h4>{ categoryName }</h4>
+                <Emoji 
+                    className="sub-category-emoji"
+                    name={ categoryName }
+                    id={phrases}
```
Comment (jdvolk) [inline src/Components/SubCategory/subCategory.js:17]: perfect

**Pull request merged**